proto/beacon/rpc/v1_gateway/BUILD.bazel

```diff
@@ -0,0 +1,16 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
```

Comment:
why are you adding this whole build file in ?

Reply:
bazel run //:gazelle -- fix made it

Reply:
removed